client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -168,20 +168,22 @@ export const getTask = (
 			taskData = {
 				timing: 10,
 				title: translate( 'Create a site menu' ),
-				description: translate(
-					"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
+				description: (
+					<>
+						{ translate(
+							"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
+						) }{ ' ' }
+						<InlineSupportLink
+							supportPostId={ 59580 }
+							supportLink={ localizeUrl( 'https://wordpress.com/support/menus/' ) }
+							showIcon={ false }
+							text={ translate( 'View tutorial.' ) }
```

Comment:
Oh, I just remembered we added analytics support to `InlineSupportLink`, so it'd be great to track some events and stats here as we currently done in [the rest of My Home](https://github.com/search?q=InlineSupportLink+repo%3AAutomattic%2Fwp-calypso+path%3Aclient%2Fmy-sites%2Fcustomer-home&type=Code&ref=advsearch&l=&l=).

```suggestion
							text={ translate( 'View tutorial.' ) }
							tracksEvent="calypso_customer_home_menus_support_page_view"
							statsGroup="calypso_customer_home"
							statsName="menus_view_tutorial"
```

Reply:
Added! :)